lms/djangoapps/courseware/tests/test_access.py

```diff
@@ -314,7 +315,12 @@ def test_has_access_in_preview_mode_with_group(self):
 
         chapter = ItemFactory.create(category="chapter", parent_location=self.course.location)
         chapter.group_access = {partition_id: [group_0_id]}
-        chapter.user_partitions = self.course.user_partitions
+
+        module_system = get_test_system()
```

Comment:
@cpennington @nasthagiri Same sort of pattern as the last test case, though here I'm using LmsPartitionService becuase the test lives in LMS.